recipes/mcap/all/conanfile.py

```diff
@@ -1,5 +1,8 @@
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools import files
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
 import os
 
 required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
New tools, new version: https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md#import-tools-from-conan